frontend/src/components/MapView/LeftPanel/AnalysisPanel/ExposureAnalysisActions/index.tsx

```diff
@@ -97,8 +138,20 @@ function ExposureAnalysisActions({
           <Typography variant="body2">{t('Download as CSV')}</Typography>
         </Button>
       )}
-      <Button className={bottomButton} onClick={handleToggleReport(true)}>
-        <Typography variant="body2">{t('Create Report')}</Typography>
+      <Button
+        id="create-report"
+        className={bottomButton}
+        onClick={handleToggleReport(true)}
+      >
+        <Typography variant="body2">{t('Preview Report')}</Typography>
+      </Button>
```

Comment:
This is using the old method, right? So very slow. Maybe we should add a warning?

`Preview Report (slow)` - wdyt?

Reply:
Yes it's the old method. I'm OK with adding (slow) label.
I will also try to add missing translations for that key